static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-international-experiment.js

```diff
@@ -18,9 +18,9 @@ define([
     mediator
 ) {
     return function () {
-        this.id = 'MembershipEngagementInternationalExperiment';
-        this.start = '2016-11-10';
-        this.expiry = '2016-12-1';
+        this.id = 'MembershipEngagementInternationalExperimentTest12';
+        this.start = '2016-12-13'; //Tues 13th Jan
+        this.expiry = '2017-1-6'; //Fri 6th Jan
```

Comment:
Yes this is intentional to give us some time to work out what to do with it after it finishes without breaking builds. From experience we tend to need this!